docs/changelog/1154.md

```diff
@@ -0,0 +1,3 @@
+- Added a NearestNeighborBase class as a parent class for the NearestNeighbor and the NearestBeighborGradient mapping
+- Added NearestNeighborGradient Mapping, an additional mapping method, that uses gradient data input from the user
```

Comment:
This sounds a bit like the mapping is already available in preCICE once this PR is merged. The changelog is not only for developers, but also for (interested) users. May I ask you to clarify a bit on this? Maybe write 'Added internals for NNG mapping implementation' or something similar.

Reply:
Yes I can do this :+1: